src/components/Panel.scss

```diff
@@ -240,3 +241,69 @@ $Panel-header-text-color: $cu-info !default;
     }
   }
 }
+
+// * 4. Collapsable Panel Rows
+
+// 1. The child row makes up the bottom border instead of parent row
+// 2. Style is nested for specificity
+// 3. The Parent row needs to be targeted seperately from a static Panel row
+
+.Panel--collapsable {
+  .Panel-parentRow { // 3
```

Comment:
Could this be a modifier instead? ie `Panel-row--parent`

Reply:
yep good call